qiskit/ignis/verification/topological_codes/fitters.py

```diff
@@ -159,13 +159,14 @@ def _make_syndrome_graph(self):
 
         return S
 
-    def get_error_probs(self, results):
+    def get_error_probs(self, results, log='0'):
```

Comment:
Could you call this to `logical`, when I first saw this I thought log file (or python logger) or logarithm. I didn't expect logical.

Reply:
Sound logical ;) Done!